packages/editor/src/components/text-editor/index.js

```diff
@@ -25,6 +26,14 @@ export default function TextEditor() {
 		};
 	}, [] );
 
+	const titleRef = useRef();
+	useEffect( () => {
+		if ( autoFocus ) {
+			return;
+		}
+		titleRef?.current?.focus();
+	}, [ autoFocus ] );
```

Comment:
This actually existed before, but I removed it inadvertently in one of the previous PRs.